core/dbt/parser/schemas.py

```diff
@@ -127,6 +128,21 @@ def resource_type(self) -> NodeType:
         return NodeType.Test
 
     def get_paths(self):
+        # TODO: In order to support this, make FilesystemSearcher accept a list
+        # of file patterns. eg: ['.yml', '.yaml']
+        yaml_files = FilesystemSearcher(
+            self.project, self.project.all_source_paths, '.yaml'
+        )
+        if yaml_files:
+            logger.warning(
+                f'We have decided that dbt release July 1, 2020 onwards,'
+                f' will start parsing core config files with `.yaml`'
+                f' extension. That means that we will continue to support'
+                f' existing `.yml` extension along with `.yaml` extension.'
+                f' You should make sure these `.yaml` files are of correct'
+                f' schema or you can choose to remove this files from the'
+                f' project.'
+            )
```

Comment:
I don't think this required the `f` prefix - there's no variable interpretation happening!

CC @drewbanin - we'll probably need your input on the exact copy to use here.

Reply:
Will remove f and update with integration test in few days, Thanks @beckjake for the pointers.

Reply:
How about some text like this for the warning?

```
A future version of dbt will parse files with both .yml and .yaml file
extensions. dbt found {len(yaml_files)} files with .yaml extensions in
your dbt project. To avoid errors when upgrading to a future release,
either remove these files from your dbt project, or change their extensions.
```